odl/discr/lp_discr.py

```diff
@@ -211,10 +211,15 @@ def is_uniform_byaxis(self):
 
     @property
     def is_uniform(self):
-        """``True`` if ``self.partition`` is uniform."""
+        """``True`` if `partition` is uniform."""
         return self.partition.is_uniform
 
     @property
+    def has_isotropic_cells(self):
```

Comment:
It's definitely less important than `is_uniform`, although related. I'll remove it then.